cfme/infrastructure/config_management.py

```diff
@@ -181,18 +417,20 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
         """
         if not force and not self.exists:
             return
-        navigate_to(self, 'All')
-        sel.check(Quadicon(self.quad_name, None).checkbox())
+        view = navigate_to(self, 'All')
+        if not view.toolbar.view_selector.is_displayed:
+            view.toolbar.refresh.click()
```

Comment:
what is it for ? If you need such reset for All page, you probably need to put it into resetter

Reply:
I was running into a problem where the view selector would disappear when the tests navigated back to that page. The refresh didn't help anyway, so I've removed that.